internal/config/config_file_test.go

```diff
@@ -121,3 +122,11 @@ func Test_parseConfigFile(t *testing.T) {
 		})
 	}
 }
+
+func Test_ReadConfigFile_error(t *testing.T) {
```

Comment:
I didn't want to submit a PR without at least one test so I added this one. I don't really like this test because it uses the actual file system. I think stubbing out the function would defeat the point as well. I couldn't think of a better way to test this functionality, but I am open to suggestions! In this case I wonder if it's worth including a test at all?